packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/new-account-form-controller.js

```diff
@@ -3,8 +3,8 @@ import find from 'lodash/find';
 import flatten from 'lodash/flatten';
 import pick from 'lodash/pick';
 import values from 'lodash/values';
-
 import { LANGUAGES } from '@ovh-ux/manager-config';
+import { getShellClient } from '../../../../shell';
```

Comment:
Can you try and inject the shell client (`shellClient`) directly in the constructor (you should then be able to use it this way: `shellClient.getShellClient()`)
It should be exposed in packages/manager/apps/dedicated/client/app/app.module.js line 236:
`.constant('shellClient', shellClient)`